redhat-cloud-client-configuration.spec

```diff
@@ -213,6 +231,138 @@ fi
 %endif
 
 
+%post cdn
+# insights-client
+%systemd_post insights-register.path
+%systemd_post insights-unregister.path
+%systemd_post insights-unregistered.path
+#rhcd
+%if 0%{?rhel} >= 8 || 0%{?fedora}
+%systemd_post rhcd.path
+%systemd_post rhcd-stop.path
```

Comment:
Will those break on newer RHELs where 'rhcd' is 'yggdrasil'?

Reply:
The `rhc-compat` [1] will take care of that.

[1] https://github.com/RedHatInsights/rhc/pull/144

Reply:
Would you mind covering that as a comment in there, please?